lib/agent/plugins/control-panel/interval/test/interval_spec.js

```diff
@@ -0,0 +1,193 @@
+var assert = require("assert"),
+    sinon  = require('sinon'),
+    should = require('should'),
+    api = require('../../api'),
+    interval = require(".."),
+    bus = require("../../bus"),
+    hooks = require('../../../../hooks'),
+    common = require('../../../../../common'),
```

Comment:
Hold on, I kept `common` because it "keeps things real". If, for any chance, we change `common` and no longers comply with what `interval` needs (not so probable, but still), then a fully faked common_obj wouldn't let us realise that there is an issue with `common`. Does it make sense?

I took https://github.com/prey/prey-node-client/blob/1.3.7-rc2/lib/agent/plugins/control-panel/test/test_push.js#L17-L21 as example, where the common_obj is built pretty much the same than here.

Also, in https://github.com/prey/prey-node-client/blob/1.3.7-rc2/lib/agent/plugins/control-panel/test/main_load.js#L22-L26 it makes sense to fake part of the common_obj because there are values that need to be manipulated.